### PR TITLE
N27 - godot-rust

### PR DESCRIPTION
**Pull request by jacobsky**
Part of #818

### CODE REVIEW

File: content/news/027/index.md [hunk 1]
```diff
@@ -204,6 +204,42 @@ This makes `hecs_rapier` feature complete, with `bevy_rapier2d` feature parity.
 [hecs_rapier]: https://github.com/smokku/hecs_rapier
 [bevy_rapier2d]: https://github.com/dimforge/bevy_rapier
 
+
+### [godot-rust](https://github.com/godot-rust/godot-rust)
```
Comment (17cupsofcoffee) [inline content/news/027/index.md:208]:
```suggestion
### [godot-rust](https://github.com/godot-rust/godot-rust)
```

File: content/news/027/index.md [hunk 2]
```diff
@@ -204,6 +204,42 @@ This makes `hecs_rapier` feature complete, with `bevy_rapier2d` feature parity.
...
+
+![godot-rust logo](godot-rust.png)
+
+godot-rust ([GitHub][gd-github], [Discord][gd-discord], [Twitter][gd-twitter])
+is a Rust library that provides bindings for the Godot game engine.
+
+Recent developments have added [foundational support to async][gd-async] 
+that enables users to make use of the Rust async runtimes with the Godot Engine
+(thanks to chitoyuu for the PR)
```
Comment (17cupsofcoffee) [inline content/news/027/index.md:217]:
```suggestion
(thanks to chitoyuu for the PR).
```

File: content/news/027/index.md [hunk 3]
```diff
@@ -204,6 +204,42 @@ This makes `hecs_rapier` feature complete, with `bevy_rapier2d` feature parity.
...
+The team also merged several smaller bug fixes in [#791][gd-791], [#795][gd-795],
+and [#800][gd-800] and is making steady progress towards version 0.10.0.
+
+Finally, the team has recently added [third party project][gd-book-projects] 
+section in the book to help promote games, applications, and libraries/tools
+that are working with godot-rust. If you have a project that you would like to
+be included, please feel free to reach out to our team and we'd be happy
+to include it.
+
+
```
Comment (17cupsofcoffee) [inline content/news/027/index.md:232]:
```suggestion

```

File: content/news/027/index.md [hunk 4]
```diff
@@ -204,6 +204,39 @@ This makes `hecs_rapier` feature complete, with `bevy_rapier2d` feature parity.
...
+The team also merged several smaller bug fixes in [#791][gd-791], [#795][gd-795],
+and [#800][gd-800] and is making steady progress towards version 0.10.0.
+
+Finally, the team has recently added [third party project][gd-book-projects] 
+section in the book to help promote games, applications, and libraries/tools
+that are working with godot-rust. If you have a project that you would like to
+be included, please feel free to reach out to our team and we'd be happy
+to include it.
+[gd-async]: https://github.com/godot-rust/godot-rust/pull/804
```
Comment (17cupsofcoffee) [inline content/news/027/index.md:230]:
```suggestion
to include it.

[gd-async]: https://github.com/godot-rust/godot-rust/pull/804
```

**Review by 17cupsofcoffee — CHANGES_REQUESTED** [review]
The content looks good, but several of the links are broken/missing URLs - can you add these?

**Comment by jacobsky** [comment]
@17cupsofcoffee Thank you for the review. Apologies for the missing links, I didn't see those when I was initially previewing the file. It is is fixed in my latest commit.

**Review by 17cupsofcoffee — APPROVED** [review]

**Pull request merged**